addons/stock/models/stock_picking.py

```diff
@@ -513,6 +513,8 @@ def _default_picking_type_id(self):
         ('name_uniq', 'unique(name, company_id)', 'Reference must be unique per company!'),
     ]
 
+    show_next_transfers = fields.Boolean(compute='_compute_show_next_transfers')
```

Comment:
```suggestion
    show_next_pickings = fields.Boolean(compute='_compute_show_next_pickings')
```
Better call them `pickings`, as it's their internal name. :wink: